src/content/applications/MyQDesktopClient.md

```diff
@@ -0,0 +1,8 @@
+---
+name: "MyQ Desktop Client"
+categories: 'Printer Driver'
```

Comment:
these are all for 'Printer Driver'
Looking at the categories, that doesnt exist. I think this should be ['printers']